src/workerd/io/compatibility-date.capnp

```diff
@@ -321,13 +321,12 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
 
   brotliContentEncoding @32 :Bool
       $compatEnableFlag("brotli_content_encoding")
+      $compatEnableDate("2023-12-01")
```

Comment:
Looks like 12/1 is a Friday. Might be worth targeting earlier in the week in case of breakages.

Reply:
This is a placeholder for now – I was thinking 12/4 should work but I'll update it to a Monday once the PR is ready.